src/schema.js

```diff
@@ -34,6 +34,7 @@ const schema = {
       ],
     },
     html_block: {
+      next: { type: 'paragraph' },
```

Comment:
You can use `previous` to indicate that the sibling node before the `html_block` is a paragraph.